test/cppsuite/test_harness/workload/database_operation.cpp

```diff
@@ -386,7 +386,7 @@ database_operation::validate(bool custom_tracking_table, const std::string &oper
     if (custom_tracking_table)
         testutil_die(EINVAL,
           "Attempting to perform default validation on a test with a user-defined tracking "
-          "table. Please define custom validation for your test.");
+          "table. Please define custom validation for your test");
```

Comment:
You didn't like the full stop?

Reply:
testutil_die appends `: Invalid Argument` to the end of the string when printing, which gave us 
`... Please define custom validation for your test.: Invalid argument`. Dropping the period looked marginally nicer